coldfront/plugins/qumulo/tests/test_tasks.py

```diff
@@ -245,3 +261,256 @@ def test_conditionally_update_storage_allocation_statuses_checks_all_pending_all
+
+        build_data = build_models()
+
+        self.project = build_data["project"]
+        self.user = build_data["user"]
+
+        self.mock_quota_response = get_mock_quota_response(
+            get_mock_quota_data(self.STORAGE2_PATH), self.STORAGE2_PATH
+        )
+
+        self.status_active = AllocationStatusChoiceFactory(name="Active")
+        self.status_ready_for_deletion = AllocationStatusChoiceFactory(
+            name="Ready for Deletion"
+        )
+
+        for index, (path, value) in enumerate(
+            get_mock_quota_data(self.STORAGE2_PATH).items()
+        ):
+            form_data = {
+                "storage_filesystem_path": path.rstrip("/"),
+                "storage_export_path": path.rstrip("/"),
+                "storage_name": f"for_tester_{index}",
+                "storage_quota": value["limit"],
+                "protocols": ["nfs"],
+                "rw_users": [f"user_{index}_rw"],
+                "ro_users": [f"user_{index}_ro"],
+                "storage_ticket": f"ITSD-{index}",
+                "cost_center": "Uncle Pennybags",
+                "department_number": "Time Travel Services",
+                "service_rate": "general",
+            }
+
+            allocation = create_allocation(
+                project=self.project, user=self.user, form_data=form_data
+            )
+            allocation.status = self.status_active
+            allocation.save()
+
+        self.storage_filesystem_path_attribute_type = (
+            AllocationAttributeType.objects.get(name="storage_filesystem_path")
+        )
+        self.storage_quota_attribute_type = AllocationAttributeType.objects.get(
+            name="storage_quota"
+        )
+
+        return super().setUp()
+
+    def tearDown(self) -> None:
+        os.environ["STORAGE2_PATH"] = self.original_storage_path
+
+        return super().tearDown()
+
+    def test_qumulo_result_set_page_limit_should_be_set(self) -> None:
+        page_limit = qumulo_api.QumuloAPI.get_result_set_page_limit()
+        self.assertIsNotNone(page_limit)
+
+    @skip("Until we have a chance to propagte the ENV variable.")
+    def test_qumulo_result_set_page_limit_should_raise_an_exception_if_not_set(
+        self,
+    ) -> None:
+        old_page_limit = os.environ.get("QUMULO_RESULT_SET_PAGE_LIMIT")
+        os.environ["QUMULO_RESULT_SET_PAGE_LIMIT"] = ""
+
+        with self.assertRaises(TypeError):
+            qumulo_api.QumuloAPI.get_result_set_page_limit()
+
+        os.environ["QUMULO_RESULT_SET_PAGE_LIMIT"] = old_page_limit
+
+    def test_after_allocation_create_usage_is_zero(self) -> None:
+        for path in get_mock_quota_data(self.STORAGE2_PATH).keys():
+            allocation_attribute_usage = None
+            try:
+                storage_filesystem_path_attribute = AllocationAttribute.objects.select_related(
+                    "allocation"
+                ).get(
+                    value=path.rstrip("/"),
```

Comment:
@harterjwustl  It is still possible to save an allocation with a trailing slash. This should work as long as the test data has paths without trailing slashes.

Reply:
Testing this, but as far as I know, you can put a trailing slash in the form, but it will be saved without, right?

Reply:
Nope, nvm.  I'll add in a few with a trailing slash to confirm behavior

Reply:
Well, it works with slashes.

@peaches7083 , do we have a stance on trailing slashes in the filesystem_path...are we going to try to go one way or another on this at some point?  The answer to the dictates if I'm going to put time into worrying about making the test data reflect that.

Reply:
Right now, any new paths are constructed by Coldfront using provided names (see `validate_relative_path`).  So they won't have a trailing slash since we don't construct them that way.

Reply:
There isn't any guarantee that *older* paths in production might not have slashes, but I don't think you need to make a special effort with test data.